ddtrace/contrib/botocore/services/kinesis.py

```diff
@@ -180,6 +185,7 @@ def patched_kinesis_api_call(original_func, instance, args, kwargs, function_var
                     ],
                 )
                 raise
+        # also what is this line??? code unreachable
         parent_ctx.end()
```

Comment:
I think this should be removed because it's now an undocumented side effect that mutates one of this function's arguments.